### PR TITLE
Added additional code needed to support the album field when construc…

### PR DESCRIPTION
**Pull request by redswitchio**
Added the additional code needed to support the album field when constructing the metadata string.

### CODE REVIEW

File: examples/ezstream-full.xml [hunk 1]
```diff
@@ -110,7 +110,13 @@
     <!-- Program to query for metadata instead of from media files -->
     <!-- <program>metadata.sh</program> -->
 
-    <!-- Metadata format -->
+    <!--
```
Comment (mgrimm) [inline examples/ezstream-full.xml]: The comments in this example files are meant as concise hints about what a setting refers to, because it's so large, and not (a copy of the) documentation. Please revert this change and update doc/ezstream.1.in.in instead.

File: src/ezstream.c [hunk 2]
```diff
@@ -158,6 +159,12 @@ buildReencodeCommand(const char *extension, const char *fileName,
 		xfree(dec_str);
 		dec_str = tmpStr;
 	}
+	if (strstr(dec_str, PLACEHOLDER_ALBUM) != NULL) {
+		char *tmpStr = replaceString(dec_str, PLACEHOLDER_ARTIST,
```
Comment (mgrimm) [inline src/ezstream.c]: That's a copypasta bug that should have been found in testing.

File: examples/ezstream-full.xml (hunk 1 above)
Comment (redswitchio) [inline examples/ezstream-full.xml]: I reverted the comment back to original concise format and updated the doc/ezstream.1.in.in with the album field information.  I left the defaults alone and just added the area that describes the placeholder and where its located in the config. I copied the format of the artist and title placeholder descriptions.

File: src/ezstream.c (hunk 2 above)
Comment (redswitchio) [inline src/ezstream.c]: My mistake, I thought I tested everything. I changed it to PLACEHOLDER_ALBUM, compiled, and test ed it.

File: examples/ezstream-full.xml (hunk 1 above)
Comment (mgrimm) [inline examples/ezstream-full.xml]: Thanks!

File: src/ezstream.c (hunk 2 above)
Comment (mgrimm) [inline src/ezstream.c]: Thanks, all taken care of!

**Comment by mgrimm** [comment]
All done, thanks! Let me know, if and how credit (e.g. in the NEWS file) should be given.

-Moritz
